bears/python/PyImportSortBear.py

```diff
@@ -182,7 +184,18 @@ def run(self, filename, file,
 
         if new_file != tuple(file):
             diff = Diff.from_string_arrays(file, new_file)
+            all_imports = ''
+            for line_nr in sorted(diff._changes):
+                linediff = diff._file[max(line_nr - 1, 0)]
```

Comment:
here too, please don't access `_file`, I think we have actually a `file` property, don't we?